docs/node_setup.md

```diff
@@ -103,7 +103,7 @@ Description=HydraDX validator
 
 [Service]
 Type=exec
-User={YOUR_SYSTEM_USER}
+User={UNIX_USER}
 ExecStart={PATH_TO_YOUR_BINARY} --chain lerna --name {YOUR_NODE_NAME} --validator
```

Comment:
here

Reply:
OK it's better to leave as-is then I don't think it's confusing